template/rust-http/Dockerfile

```diff
@@ -27,12 +27,12 @@ RUN mkdir -p /home/app \
 
 WORKDIR /home/app
 
-COPY --from=build --chown=app:app /home/rust/main/target/x86_64-unknown-linux-musl/release/main .
+COPY --from=build --chown=app:app /home/rust/main/target/x86_64-unknown-linux-musl/release/main /usr/bin/main
 
 USER app
 
 # Set up watchdog for HTTP mode
-ENV fprocess="./main"
+ENV fprocess="main"
```

Comment:
I noticed the fix for https://github.com/openfaas-incubator/rust-http-template/issues/10 was mentioned in https://github.com/openfaas-incubator/rust-http-template/pull/6

For some reason I don't understand, the watchdog fprocess works if the binary is available on the path thus copying to `/usr/bin/main` instead of the app home directory.

I also tried with the absolute path `ENV fprocess="/home/app/main"`

With the same result as the relative: `./main`

Reply:
We would generally put the handler/main binary in the home directory and not into /usr/bin.

Reply:
That would be the best for sure. I am having trouble finding where the issue lies. For some reason it works with the binary in the home directory if I spin the image up through docker or crt, but it crashes with:

```
2023-03-28T17:04:54Z 2023/03/28 17:04:54 Error scanning stderr: read |0: file already closed
2023-03-28T17:04:54Z 2023/03/28 17:04:54 Error scanning stdout: read |0: file already closed
2023-03-28T17:04:54Z 2023/03/28 17:04:54 Forked function has terminated: exec: not started
```

When running in OpenFaaS (faasd and CE)

Reply:
I'll give it a try when I can to compare